templates/settings/_listFormatEvent.html.twig

```diff
@@ -5,18 +5,32 @@
     </div>
 </div>
 
-<div class="row mt-3">
+<div class="row my-3">
+
     {% if formatEvents is defined and formatEvents|length > 0 %}
         {% for formatEvent in formatEvents %}
```

Comment:
tu peux je pense enlever ton if au dessus du for et utiliser la syntaxe de twig
{% for www in xxx %}

{% else %}
  Aucun format...
{%endfor%}

Reply:
Ok correction effectuée